### PR TITLE
docs: Add README template

### PR DESCRIPTION
**Pull request by badr4php**
Related to Jira: [ENG-1450]

[ENG-1450]: https://comicrelief.atlassian.net/browse/ENG-1450?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.Template.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Project Name [![CircleCI](https://circleci.com/gh/comicrelief/serverless-payments.svg?style=svg&circle-token=f65be9c11820771b57a54e836db98d22d4ad2fc2)](https://circleci.com/gh/comicrelief/serverless-payments)
```
Comment (ahmedelbougha) [inline README.Template.md:1]: I think the build status should be using github instead of CircleCI

**Comment by badr4php** [comment]
@zhibek @seb-cr can you review this branch?

**Review by seb-cr — COMMENTED** [review]
Good start, but I think this template is too empty.

Have a look at the project readmes for [serverless-giftaid](https://github.com/comicrelief/serverless-giftaid) and [serverless-marketing-preferences](https://github.com/comicrelief/serverless-marketing-preferences). Most projects are now very similar to those, and it makes going between projects easier if they're all the same.

Some general suggestions:

- copy readme structure from the above repos
  - in particular, I think the most important section is how to run feature tests
- add prompts for what to write in descriptions. For example, for testing:
  - do you need AWS credentials? which account?
  - what environment variables need to be set?
  - can you test using `serverless-offline` or LocalStack?

**Review by pauldolden — APPROVED** [review]
All looks good to me!

**Pull request merged**